src/test/kotlin/study/CalculatorTest.kt

```diff
@@ -0,0 +1,122 @@
+package study
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import java.util.stream.Stream
+
+class CalculatorTest {
+    @ParameterizedTest
+    @MethodSource("arrangePlusTest")
+    fun 덧셈(input: String, expected: Double) {
```

Comment:
```suggestion
    fun `덧셈`(input: String, expected: Double) {
```
함수명을 한글로 표현한다면 위처럼 선언할수도 있습니다 😄